Rollbar/RollbarNotifier.m

```diff
@@ -292,15 +305,23 @@ - (NSDictionary*)buildClientData {
     }
 }
 
-- (NSDictionary*)buildPayloadWithLevel:(NSString*)level message:(NSString*)message exception:(NSException*)exception extra:(NSDictionary*)extra crashReport:(NSString*)crashReport context:(NSString*)context {
+- (NSDictionary*)buildPayloadWithLevel:(NSString*)level
+                               message:(NSString*)message
+                             exception:(NSException*)exception
+                                 extra:(NSDictionary*)extra
+                           crashReport:(NSString*)crashReport
+                               context:(NSString*)context {
     
     NSDictionary *clientData = [self buildClientData];
     NSDictionary *notifierData = @{@"name": self.configuration.notifierName,
                                    @"version": self.configuration.notifierVersion};
     
     NSDictionary *customData = self.configuration.customData;
     
-    NSDictionary *body = [self buildPayloadBodyWithMessage:message exception:exception extra:extra crashReport:crashReport];
+    NSDictionary *body = [self buildPayloadBodyWithMessage:message
+                                                 exception:exception
+                                                     extra:extra crashReport:crashReport
```

Comment:
Well if we are going to make all of these formatting changes then should also put crashReport on the next line

Reply:
i definitely missed this one. thanks!